pkg/karmadactl/options/global.go

```diff
@@ -27,3 +28,10 @@ func (o *GlobalCommandOptions) AddFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.KubeConfig, "kubeconfig", "", "Path to the control plane kubeconfig file.")
 	flags.StringVar(&o.KarmadaContext, "karmada-context", "", "Name of the cluster context in control plane kubeconfig file.")
 }
+
+// AddFlags adds flags to the specified FlagSet.
+func AddFlags(cf *genericclioptions.ConfigFlags, flags *pflag.FlagSet) {
+	flags.StringVar(cf.KubeConfig, "kubeconfig", *cf.KubeConfig, "Path to the kubeconfig file to use for CLI requests.")
+	flags.StringVar(cf.Context, "karmada-context", *cf.Context, "The name of the kubeconfig context to use")
+	flags.StringVarP(cf.Namespace, "namespace", "n", *cf.Namespace, "If present, the namespace scope for this CLI request")
```

Comment:
The `namespace` is not general, like `cordon`, `join`, `token` command etc, so I suggest to only retain `kubeconfig` and `karmada-context`.

Reply:
fixed this and `taint` @lonelyCZ